### ISSUE TITLE
Allow aria-label and aria-labelledby attributes on all input components.

### ISSUE THREAD

**Issue by efc** [primefaces/primereact#839]
```
[ ] bug report
[X] feature request
[ ] support request
```

**Current behavior**

I cannot associate `aria-label` or `aria-labelledby` attributes with `Dropdown` or other PrimeReact input components. (This was noted in issue #729, but that issue was closed without addressing this request.)

**Expected behavior**

I would expect that I could define a `Dropdown` and any other PrimeReact input components like this:

```
<Dropdown value='test' editable={true} options={[]} ariaLabel='Test' />
```

or

```
<Dropdown value='test' editable={true} options={[]} ariaLabelledby='testid' />
```

* **React version:**
16.8.6

* **PrimeReact version:**
3.1.2

* **Browser:**
all
 
* **Language:**
all

**Comment by mertsincan** [comment]
Components;
- [x] InputText
- [x] InputTextarea
- [x] DropDown https://github.com/primefaces/primereact/issues/855
- [ ] Calendar
- [ ] MultiSelect

**Comment by arnauddem** [comment]
InputSwitch should also allow aria attributes

**Comment by ankitprahladsoni** [comment]
Will you add this support for `InputNumber` as well? Right now it only has `aria-labelledby` attribute.

**Comment by jeffgaynorspok** [comment]
`aria-*` for accessibility and `data-*` for testing should be passed through on the root of all components.

**Comment by melloware** [comment]
@jeffgaynorspok this feature was added in this ticket: https://github.com/primefaces/primereact/issues/2620
